agent/bench-scripts/pbench-fio

```diff
@@ -426,14 +426,16 @@ function fio_install() {
 # install python-pandas on the controller: fiologparser_hist.py needs it
 function pandas_install() {
 	if [ "$postprocess_only" = "n" ]; then
-		if check_install_rpm python-pandas; then
+	    if check_install_rpm python2-pandas; then
+			debug_log "[$script_name]python2-pandas is installed"
```

Comment:
Let's just emit `python-pandas` instead of `python2-pandas` as it should not matter that much and makes this easier to get correct for unit tests.